ray-operator/apis/ray/v1alpha1/events/raycluster_events.go

```diff
@@ -0,0 +1,5 @@
+package v1alpha1
```

Comment:
lmk if this file path is OK

Reply:
From the this change, I didn't see real values and I am wondering if that would be good for your customization in downstream later?

Reply:
I was hoping some string constants would help people adding K8s event reasons in the future. They can select from a centralized list of reason strings. Chance of arbitrary reasons can be reduced.

I originally wanted to put this in `ray-operator/apis/ray/v1alpha1/raycluster_types.go`, but the `Unhealthy` const conflicted with the existing `Unhealthy ClusterState` const. Now that the const is renamed to `RayConfigError`, I can put this entire block in that file like so.

```go
type EventReason string

const (
	RayConfigError EventReason = "RayConfigError"
)
```

thoughts?

Reply:
see second commit

Reply:
Great. I actually prefer current way, Once we have enough events reason, we can consider to create separate package. Let's keep the simplicity at this moment.